R/rdataretriever.R

```diff
@@ -136,9 +137,11 @@ fetch = function(dataset, quiet=TRUE, data_names=NULL){
         }
         bone[[i]] = tempdata
     }
+    if(length(bone)>0){
     if (length(bone) == 1)
```

Comment:
Come code refactoring/indentation may be needed here

Reply:
Sure ! I will indent it properly